### DISCUSSION TITLE
npx @upptime/uptime-monitor summary summary gives error

### DISCUSSION THREAD

**Discussion by xiaoas** [upptime/upptime#483] [Q&A]
I tried running:

```sh
$ npx @upptime/uptime-monitor summary
```
documented in https://github.com/upptime/uptime-monitor#readme, and was given the following error:

npm ERR! could not determine executable to run

**Comment by artemmukhin** [comment]
I am facing the same issue. @AnandChowdhary could you please elaborate?
